.github/codeql.yaml

```diff
@@ -0,0 +1,6 @@
+paths-ignore:
+  - src/pkg/packager/network.go
```

Comment:
Yeah it's always going to trigger on new commits for some reason. I've dismissed the same warning a dozen times and those will always have issues in that file.  I think this is okay for now since we weren't checking anything at all just a few weeks ago and this is a small file that will have known red flags for scanner.